roles/kubernetes/secrets/files/.gitignore

```diff
@@ -0,0 +1,4 @@
+tokens/*.token
+tokens/known_tokens.csv
+certs/*.pem
+openssl.conf
```

Comment:
This .gitignore isn't necessary since the latest commit.
Indeed the secrets are now generated directly into the first master.